.gitignore

```diff
@@ -1 +1,2 @@
 /target
```

Comment:
This is not the correct gitignore, please use this https://www.toptal.com/developers/gitignore/api/rust and remove thte Cargo.lock file from the repo

Reply:
Done. The Cargo.lock is now in the .gitignore